data-access/nexustiles/backends/zarr/backend.py

```diff
@@ -42,7 +42,8 @@
 class ZarrBackend(AbstractTileService):
     def __init__(self, dataset_name, path, config=None):
         AbstractTileService.__init__(self, dataset_name)
-        self.__config = config if config is not None else {}
```

Comment:
Just a suggestion. 
The default `config` parameter can be set to `{}`.
Then you don't need to validate it.

Reply:
If I do that my IDE complains. Either way is fine by me, I just find the IDE warnings annoying